packages/generator/src/edmx-to-vdm/v4/action-import.spec.ts

```diff
@@ -0,0 +1,239 @@
+import { EdmxParameter, EdmxProperty } from '../../edmx-parser/common';
+import {
+  EdmxAction,
+  EdmxActionImport,
+  EdmxComplexType,
+  EdmxEntitySet,
+  EdmxEntityType
+} from '../../edmx-parser/v4';
+import { ServiceMetadata } from '../../edmx-parser/edmx-file-reader';
+import { generateEntitiesV4 } from './entity';
+import { generateActionImportsV4 } from './action-import';
+import { generateComplexTypesV4 } from './complex-type';
+import { createEntityType, getComplexType, getFormatter } from './entity.spec';
+
+describe('action-import', () => {
```

Comment:
I assume this this is an old test that has been moved from the new `packages/generator/src/edmx-to-vdm/v4/entity.spec.ts`